cypress/e2e/shoppingBag_spec.cy.js

```diff
@@ -0,0 +1,69 @@
+describe(('Shopping Bag Page'), () => {
+  beforeEach(() => {
+    cy.intercept('POST', 'https://everuse-be-b6017dbfcc94.herokuapp.com/', {
+      statusCode: 201,
+      fixture: 'mockBag'
```

Comment:
I imagine this user flow will have to change a little bit when we start shoppingBag off as an empty array instead of the mock data we have initially rn

maybe there is a way we can do a custom command to fill the bag?

Reply:
I think you're right. That's a great idea. I love commands!

Reply:
Agreed, for now it will work, but once we get all the pieces merged will probably have to refactor.

Nice stubbing! Thanks for the beta!